vcell-rest/src/test/java/org/vcell/restq/apiclient/UsersApiTest.java

```diff
@@ -178,6 +189,27 @@ public void testOldAPITokenGenerationForGuest() throws ApiException {
         Assertions.assertEquals("140220477", guestApiToken.getUserKey());
     }
 
+    @Test
+    public void testUserMiddleWare() throws ApiException{
+        ApiClient defaultUser = TestEndpointUtils.createUnAuthenticatedAPIClient(testPort);
+        PublicationResourceApi publicationResourceApi = new PublicationResourceApi(defaultUser);
+
+        // public available to everyone
+        Assertions.assertDoesNotThrow(publicationResourceApi::getPublications);
+
+        // guests can not delete a publication
```

Comment:
anonymous users (or unauthenticated calls) can not delete a publication.  Note that there are additional roles necessary for deleting publications.